GDE-21: Form styles

steveworley opened the pull request:
- Adds theme hook suggestions in base template.php to add per element type label and element themes.
- Adds theme elements to produce correct markup for uikit.
- Adds additional classes to existing elements to render as expected.

steveworley committed:
GDE-21: Add templates for form elements for uikit.

- Adds theme hook suggestions in base template.php to add per element type label and element themes.
- Adds theme elements to produce correct markup for uikit.
- Adds additional classes to existing elements to render as expected.

steveworley committed:
GDE-21: Form inline-containers for uikit.

- Adds a custom form.scss file to the theme so we can provide inline elements as per Drupal configuration.

steveworley committed:
Merge branch 'master' into feature/gde-21-form-styles

steveworley committed:
GDE-21: Add radio, button and select templates.

- Adds different form element tempaltes to match markup to what is expected by UI Kit.

steveworley committed:
GDE-21: Remove empty webform template file.

- Removes the empty webform template file.

yusufhm commented on docroot/themes/custom/govcms_uikit_starterkit/govcms_uikit_starterkit.theme:
```diff
@@ -145,3 +145,10 @@ function govcms_uikit_starterkit_preprocess_menu(&$variables) {
+/**
+ * Implements hook_preprocess_form_element().
+ */
+function govcms_uikit_starterkit_preprocess_form_element(&$variables) {
```
@steveworley You have the same code in `docroot/themes/custom/govcms_uikit_base/STARTERKIT/theme-settings.php`. Is it intentional? It seems like it should have been in `STARTERKIT/govcms_uikit_starterkit.theme` instead.

steveworley replied:
Whoops!

steveworley committed:
GDE-21: Move starterkit hook into correct file.

- Updates the startkit theme and moves the form element hook in the theme .theme file rather than the settings file.
